G/gh_cli/build_tarballs.jl

```diff
@@ -3,28 +3,28 @@
 using BinaryBuilder, Pkg
 
 name = "gh_cli"
-version = v"1.0.0"
+version = v"2.2.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/cli/cli.git", "b2e36a0979a06b94bf364552a856c166cd415234"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/cli/cli.git", "a843cbd72813025817a2293a09b31c4597a3f655"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/cli/
-# Use newer termenv to work around issue while building for FreeBSD:
-# https://github.com/muesli/termenv/issues/17
-atomic_patch -p1 ../patches/use-termenv-v0.7.2.patch
-make
+
+export GO_LDFLAGS="-s -w"
```

Comment:
Are you sure this is really needed?  We usually avoid stripping if possible, patchelf sometimes doesn't like it (even though here it didn't complain)

Reply:
Dropped it, rerunning